example/spec/parallel/kinesisTests/KinesisReplaySpec.js

```diff
@@ -166,33 +167,37 @@ describe('The Kinesis Replay API', () => {
     });
 
     describe('processes messages within the specified time slice', () => {
+      let workflowArn;
+
+      beforeAll(async () => {
+        workflowArn = await getWorkflowArn(testConfig.stackName, testConfig.bucket, rules[0].workflow);
+      });
+
       it('to start the expected workflows', async () => {
         console.log('Waiting for step functions to start...');
-        const expectedWorkflows = targetedRecords.map((record) => waitForAllTestSf(
+        const expectedWorkflows = targetedRecords.map((record) => waitForAllTestSfForRecord(
```

Comment:
Couldn't you use ` waitForTestSfForRecord` here and in the other cases where you're expecting 1 execution?

Reply:
Fair point. Will update.

Reply:
1432b17